shared/rakefiles/xk_infra.rake

```diff
@@ -97,3 +107,11 @@ task :apply_common_infra => [@gcp_creds_file] do
     sh "#{@exekube_cmd} gsutil versioning set on gs://#{ENV["TF_VAR_project_id"]}-tfstate"
   end
 end
+
+task :fix_common_service_account_permissions => [@gcp_creds_file] do
```

Comment:
I don't really like having two tasks with the same name in different rake files, feels a little bit confusing. But since this is a side effect of having part of the code that runs inside of the xk container, I don't have a better proposition at the moment...